notebooks/GraphNotebook.ipynb

```diff
@@ -544,9 +600,14 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.10.5"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "b0fa6594d8f4cbf19f97940f81e996739fb7646882a419484c72d19e05852a7e"
+   }
```

Comment:
Not sure if these changes to the notebook are intended.

Reply:
No, they are not - thanks for the review!